applications/trilinos_application/external_includes/epetra_default_setter.h

```diff
@@ -0,0 +1,108 @@
+//    |  /           |
```

Comment:
unfortunately it shows this file as new file, even though I renamed it and then cleaned it.
Good thing it contains almost nothing :)